src/lib/geo-location.js

```diff
@@ -43,7 +43,7 @@ const getGeoLocation = async ({ geoCountry, mode, offline, state }) => {
 
   // If we have cached geolocation data and the `--geo` option is set to
   // `cache`, let's try to use it.
-  if (cacheObject !== undefined && mode === 'cache') {
+  if (cacheObject !== undefined && mode === 'cache' && !geoCountry) {
```

Comment:
I'm not sure there's ever any situation where you'd rather receive dummy data than an actual location from the country you're trying to emulate. But I'll defer to you.

Reply:
No, after thinking about it, I agree with you.  I think I was originally looking down the road a bit more towards mocking subdivisions/states at which point the caching might be counterproductive.  But seeing as that is probably either not going to happen, or a ways off at this point, I think your approach makes sense.  I'll abstract out that logic into a function of its own, and it can be modified later if needed.